src/Button/Button.docs.json

```diff
@@ -0,0 +1,70 @@
+{
+  "id": "button",
+  "name": "Button",
+  "status": "alpha",
+  "a11yReviewed": false,
```

Comment:
is it possible we can apply a logic that indicates when the component was last reviewed? if components will be changing often, which I hope they will be, we'll want to indicate how long it's been since we've approved it. I think!

Reply:
👋🏼 @andrialexandrou, sorry for late response! We have [this issue](https://github.com/github/accessibility/issues/789) in our backlog to track of the last a11y reviewed SHA of the components. Is this something similar what you were thinking? I don't think we have discussed what kind of automation logic we could implement for[this issue](https://github.com/github/accessibility/issues/789) yet. Do you think DataDog would be more convenient for accessibility team to see the accessibility data or do you think maybe we could have this data in multiple places?

Reply:
this information would be useful for everyone if it could be surfaced in the Primer docs somehow. it's okay to defer on this just didn't know latest decisions.

Reply:
No worries! We will catch up about how to solve this, hopefully soon!